### PR TITLE
Fix/ Added animation on tooltip, dropdown, pages and modal

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
Added animation on tooltip, dropdown, pages and modal

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@app/pages/Collections/CollectionsPage.svelte [hunk 1]
```diff
@@ -324,6 +329,16 @@
...
+<ModalWrapperV1
+  title={"New Collection"}
+  type={"dark"}
+  width={"35%"}
+  zIndex={1000}
+  isOpen={isImportCollectionPopup}
+  handleModalState={(flag = false) => {
+    isImportCollectionPopup = flag;
+  }}
+>
```
Comment (Astitva877) [inline src/packages/@app/pages/Collections/CollectionsPage.svelte:341]: Why we have changed the structure of import collection, i think we have just put the modal wrapper outside, any specific reason for that?

File: src/packages/@app/pages/Collections/CollectionsPage.svelte [hunk 2]
```diff
@@ -381,16 +396,27 @@
...
+<ModalWrapperV1
+  title={"Import cURL"}
+  type={"dark"}
+  width={"35%"}
+  zIndex={1000}
+  isOpen={isImportCurlPopup}
+  handleModalState={(flag = false) => {
+    isImportCurlPopup = flag;
+  }}
+>
```
Comment (Astitva877) [inline src/packages/@app/pages/Collections/CollectionsPage.svelte:411]: Same we did for import curl why?

File: src/packages/@app/pages/Collections/CollectionsPage.svelte (hunk 1 above)
Comment (itsmdasifraza) [reply src/packages/@app/pages/Collections/CollectionsPage.svelte:341]: this is to handle re-renders that invokes animation every time modal opens

File: src/packages/@app/pages/Collections/CollectionsPage.svelte (hunk 2 above)
Comment (itsmdasifraza) [reply src/packages/@app/pages/Collections/CollectionsPage.svelte:411]: this is to handle re-renders that invokes animation every time modal opens

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**